site/docs/getting-started/develop.mdx

```diff
@@ -3,10 +3,6 @@ sidebar_label: Develop
 hide_table_of_contents: true
 ---
 
-import Link from "@docusaurus/Link";
```

Comment:
FYI: These were redundant imports, so I took this opportunity to strip them out (I suspect VSCode auto-complete may have been overzealous and added them at some point due to the _example code_ on this page).

Reply:
I think this is required for the link on line 152

Reply:
I think so too

Reply:
Oops. Looks like I got a bit trigger happy. Just pushed a fix for this